src/main/javac/org/apache/brooklyn/tosca/a4c/platform/transformer/converters/ToscaTomcatServerConverter.java

```diff
@@ -0,0 +1,74 @@
+package org.apache.brooklyn.tosca.a4c.platform.transformer.converters;
+
+import alien4cloud.model.components.AbstractPropertyValue;
+import alien4cloud.model.components.ScalarPropertyValue;
+import alien4cloud.model.topology.NodeTemplate;
+import com.google.common.collect.ImmutableList;
+import org.apache.brooklyn.api.entity.EntitySpec;
+import org.apache.brooklyn.api.mgmt.ManagementContext;
+import org.apache.brooklyn.core.location.PortRanges;
+import org.apache.brooklyn.entity.webapp.jboss.JBoss7Server;
+import org.apache.brooklyn.entity.webapp.tomcat.TomcatServer;
+import org.apache.brooklyn.util.text.Strings;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.Map;
+
+public class ToscaTomcatServerConverter {
+
```

Comment:
This converter allows to create a EntitSpec<TomcatServer> from a `tosca.nodes.Tomcat` template.
However, the custom type `tosca.nodes.Tomcat` is not the best option. As I understood previously, we should use custom-types for representing the Brooklyn entities.